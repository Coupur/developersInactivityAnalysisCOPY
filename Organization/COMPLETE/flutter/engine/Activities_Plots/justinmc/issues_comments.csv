id,date,creator_login
501905632,2019-06-13 22:47:19,justinmc
502293054,2019-06-14 21:58:24,justinmc
502302561,2019-06-14 22:44:26,justinmc
488834815,2019-05-02 21:14:50,justinmc
483816525,2019-04-16 19:45:28,justinmc
483851719,2019-04-16 21:32:03,justinmc
484245870,2019-04-17 20:20:33,justinmc
481840923,2019-04-10 20:02:17,justinmc
482699353,2019-04-12 19:44:00,justinmc
483365786,2019-04-15 18:28:49,justinmc
484182427,2019-04-17 17:18:16,justinmc
484293047,2019-04-17 23:01:51,justinmc
475399178,2019-03-21 20:56:31,justinmc
452915638,2019-01-09 23:47:01,justinmc
449495976,2018-12-21 20:54:42,justinmc
450011197,2018-12-26 19:11:57,justinmc
453610689,2019-01-11 18:23:45,justinmc
448033318,2018-12-17 23:07:53,justinmc
448775751,2018-12-19 22:57:02,justinmc
449149183,2018-12-20 22:00:36,justinmc
449174157,2018-12-20 23:31:32,justinmc
451795200,2019-01-07 01:20:43,justinmc
442555946,2018-11-28 18:34:21,justinmc
444300349,2018-12-04 23:33:08,justinmc
