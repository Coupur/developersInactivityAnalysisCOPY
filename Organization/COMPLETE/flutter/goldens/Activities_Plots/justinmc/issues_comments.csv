id,date,creator_login
512860939,2019-07-18 15:17:19,justinmc
494538992,2019-05-21 20:08:22,justinmc
